### PR TITLE
Fix Enum value comparison for python v3.8

### PR DESCRIPTION
**Pull request by jmyatt**
A change to Enum's __ contains __ method in python v3.8 causes a TypeError in a value comparison check.  This fixes a current instance of this error when sending sailthru emails.

I could not find any other Enum usage in this repo that would hit the same issue

### CODE REVIEW

**Review by mikix — APPROVED** [review]
Sure. Why was this being run under py3.9?

**Comment by jmyatt** [comment]
oops I meant v3.8.  comment fixed now

**Comment by cpennington** [comment]
Why did this ever work?

**Comment by cpennington** [comment]
Ah: In short, it never did, but it did it silently: https://github.com/python/cpython/blob/3.5/Lib/enum.py#L246

`Enum.__contains__` in 3.5 was
```
    def __contains__(cls, member):
        return isinstance(member, cls) and member._name_ in cls._member_map_
```

Since the error codes weren't enum instances, this check always return `False`, without complaint.

Using the by-value lookup (`RecoverableErrorCodes(error_code)`) and then catching the `ValueError` might be a clearer way to do this lookup?

**Comment by jmyatt** [comment]
yea the type check and subsequent TypeError was added after 3.5: https://github.com/python/cpython/blob/3.8/Lib/enum.py#L321

so as you said, this comparison was failing silently before.  I think the solution in this PR makes sense in this case.  Are you suggesting a change that should be made to python's `enum` implementation?

**Comment by cpennington** [comment]
No, I was thinking about a possible alternate implementation of `error_code in [rec.value for rec in RecoverableErrorCodes]`, but I'm not sure it helps. That said, as I look further down, we check `error_code == RecoverableErrorCodes.RATE_LIMIT` which won't work because `RecoverableErrorCodes` isn't an `IntEnum`.

**Comment by jmyatt** [comment]
ah yes good point - I'll put up a change for that

File: CHANGELOG.rst [hunk 1]
```diff
@@ -16,6 +16,11 @@ Unreleased
 
 *
 
+[0.1.16] - 2020-10-17
+~~~~~~~~~~~~~~~~~~~~~
+
+* Fixed enum usage for Python 3.9
```
Comment (nedbat) [inline CHANGELOG.rst:22]: Was there some consumer-visible error that this fixed?  It would be good to note it in the changelog.

Comment (jmyatt) [reply CHANGELOG.rst:22]: ok I'll alter that note with that info

**Pull request merged**